vl-kontrakt-soeknad/soeknad-v3/src/main/resources/xsd/uttak/uttak-v3.xsd

```diff
@@ -75,6 +75,7 @@
 	<xsd:complexType name="Fordeling">
 		<xsd:sequence>
 			<xsd:element name="annenForelderErInformert" type="xsd:boolean" />
+			<xsd:element name="oenskerJustertVedFoedsel" minOccurs="0" type="xsd:boolean" />
```

Comment:
litt vel generelt navn? eller går man for alltid true for mor?

Reply:
Tenker det er greit slik. Det er forskjell på "justeringen" vi gjør på mor, vi innvilger ikke tidsperioder hun ikke har søkt om